### PR TITLE
added archetype for generating simple project

### PR DESCRIPTION
**Pull request by DariaMitrofanovna**

### CODE REVIEW

File: tesler-archetypes/tesler-simple-archetype/src/main/resources/archetype-resources/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Global Network Standards and Audits
```
Comment (Dergash) [inline tesler-archetypes/tesler-simple-archetype/src/main/resources/archetype-resources/README.md]: Does not belong here

File: tesler-archetypes/tesler-simple-archetype/pom.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
```
Comment (abratashev) [inline tesler-archetypes/tesler-simple-archetype/pom.xml:3]: I think we need to add this pom as <module> in parent pom to compile it from root

File: tesler-core/src/main/java/io/tesler/core/metahotreload/controller/MetaHotReloadController.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package io.tesler.core.metahotreload.controller;
+
+import io.tesler.core.metahotreload.service.MetaHotReloadService;
```
Comment (abratashev) [inline tesler-core/src/main/java/io/tesler/core/metahotreload/controller/MetaHotReloadController.java]: These classes relate to https://github.com/tesler-platform/tesler/pull/146 , right?

**Comment by abratashev** [comment]
I tried to generate archetype - it creates "ui" folder near the pom project. It's not сorrect, is it?

image

**Review by abratashev — CHANGES_REQUESTED** [review]
Please fix requests above

File: tesler-archetypes/tesler-simple-archetype/src/main/resources/archetype-resources/README.md (hunk 1 above)
Comment (DariaMitrofanovna) [reply tesler-archetypes/tesler-simple-archetype/src/main/resources/archetype-resources/README.md]: fixed

File: tesler-archetypes/tesler-simple-archetype/pom.xml (hunk 2 above)
Comment (DariaMitrofanovna) [reply tesler-archetypes/tesler-simple-archetype/pom.xml:3]: fixed

File: tesler-core/src/main/java/io/tesler/core/metahotreload/controller/MetaHotReloadController.java (hunk 3 above)
Comment (DariaMitrofanovna) [reply tesler-core/src/main/java/io/tesler/core/metahotreload/controller/MetaHotReloadController.java]: fixed

**Comment by DariaMitrofanovna** [comment]
> I tried to generate archetype - it creates "ui" folder near the pom project. It's not сorrect, is it?
> 
> image

fixed

**Pull request merged**